src/style.scss

```diff
@@ -0,0 +1,135 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
When I remove the height, the page does not break, it is necessary that the height depends on the content
<img width="1466" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/49218b4b-146e-46aa-b430-7026644e2d6d">